tests/functional/replication/queueProcessor.js

```diff
@@ -843,7 +843,7 @@ describe('queue processor functional tests with mocking', () => {
                           objectSizeMetrics: [100, 1000],
                         },
                         {},
-                        { topic: 'metrics-test-topic' });
+                        { topic: 'metrics-test-topic' }, { topic: 'backbeat-gc' });
```

Comment:
Nit: consistency with other lines

```suggestion
                        { topic: 'metrics-test-topic' },
                        { topic: 'backbeat-gc' });
```